src/Parts/MyTrainingCard/SingleTrainingCard.jsx

```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+/* eslint-disable react/prop-types */
```

Comment:
Please remove the fixing comment and make sure use current linter

Reply:
already fix eca2dd60b6bc983ef310fad67efbb388a63ba38f